.github/workflows/try-release-daily.yml

```diff
@@ -0,0 +1,69 @@
+name: Try building a release daily
+# Build a release tarball for all supported branches to verify the code is releasable.
+
+on:
+  schedule:
+    - cron: 0 3 * * *
+  # be able to start this action manually from a actions tab when needed
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  daily-release-test:
+    runs-on: [self-hosted, kstest]
+    strategy:
+      fail-fast: false
+      matrix:
+        branch: ['master', 'rhel-8', 'rhel-9'] # depending on season, also: fNN-release
+
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ matrix.branch }}
+
+      - name: Build anaconda container (to make the release)
+        run: |
+          make -f Makefile.am anaconda-release-build
+
+      - name: Run the build in the container
+        run: |
+          make -f Makefile.am container-release | tee release.log
+
+      - name: Check logs for malformed translations
+        continue-on-error: true
+        run: |
+          errors=$(grep -E "Unable to compile [^:]+: .*" release.log || true)  # grep exits with 1 if no lines found
+          if [ -n "$errors" ]; then
+            echo "Failed to compile some translations:"
+            echo "$errors"
+            exit 1
+          else
+            echo "No translation compilation errors found."
+          fi
+
+      - name: Check if translation files are present in tarball
+        continue-on-error: true
+        run: |
+          tarfiles=$(tar -tf anaconda-*.tar.bz2)
+          missing=0
+          required_languages="fr ja zh_CN es pt de ru ar"
+          # This list comes from langtable's list_common_languages, which in turn reportedly comes
```

Comment:
If we wanted to run a RHEL specific version of this, lnovy might have a translation priority list for RHEL.

Reply:
ljanda says: use languages in a bullet list here: https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/8.6_release_notes/index#international_languages